lib/puppet/functions/archive/go_md5.rb

```diff
@@ -0,0 +1,32 @@
+# Public: go file md5 checksum
+#
+# args[0] - username
+# args[1] - password
+# args[2] - file_name
+# args[3] - go md5 checksum url
```

Comment:
Could you convert this documentation to the new puppet strings format?  eg like https://github.com/voxpupuli/puppet-archive/blob/ab996a12193497c13c3a2e151915652289cd1e0b/lib/puppet/functions/archive/artifactory_checksum.rb#L5-L9